adapterbox/install.sh

```diff
@@ -0,0 +1,10 @@
+sudo apt-get --yes install libcurl3 php7.0-curl php7.0-cli php7.0-xml
+sudo usermod -a -G lpadmin pi
+sudo systemctl enable ssh
+wget http://www.wawision.de/adapterbox.deb 
+sudo dpkg -i adapterbox.deb
+echo "Seriennummer eingeben gefolgt von einem [ENTER]:"
+read seriennummer
+echo "<?php \$config['serial'] = '$seriennummer'; ?>" > /tmp/config.php
+sudo mv /tmp/config.php /root/config.php
```

Comment:
```suggestion
sudo mv -f /tmp/config.php /root/config.php
```
In case the file this should override it

Reply:
this PR should only make the old files available. Improving the code will be another PR.